task.go

```diff
@@ -447,6 +446,7 @@ func (e *Executor) GetTask(call *ast.Call) (*ast.Task, error) {
 			call.Vars = &ast.Vars{}
 		}
 		call.Vars.Set("MATCH", ast.Var{Value: matchingTasks[0].Wildcards})
+		matchingTasks[0].Task.FullName = call.Task
```

Comment:
It looks like we'd be getting a race condition here, no? Parallel calls to this task would be overiding the same attribute.

Tricky to implement this in another way, though, I know...

Reply:
You are totally right. The same task with a wildcard, used twice in a dep (ran in parallel) the first one will be overridden by the last

Put this in draft until I find a solution for it

Thanks!